dags/client_count_daily.py

```diff
@@ -0,0 +1,26 @@
+from airflow import DAG
+from datetime import datetime, timedelta
+from operators.emr_spark_operator import EMRSparkOperator
+
+default_args = {
+    'owner': 'frank@mozilla.com',
+    'depends_on_past': False,
+    'start_date': datetime(2016, 8, 13),
```

Comment:
Let's set this to a recent start date, then do backfill separately. I think we may want to add a couple more columns before we go back in time, so let's change this to `datetime(2016, 11, 1)`.